samples/charts/data-chart/radial-series/Pages/DataChartRadialSeries.razor

```diff
@@ -0,0 +1,193 @@
+        <select value=@SeriesType
+                @onchange="OnSeriesTypeChanged">
+            <option>Area</option>
+            <option>Column</option>
+            <option>Pie</option>
+            <option>Line</option>
+        </select>
+
+        <span class="igLegend-title">Legend: </span>
+        <div class="igLegend">
+            <Legend @ref="LegendRef" Orientation="LegendOrientation.Horizontal" />
+        </div>
+    </div>
+
+    <div class="igComponent">
+        @if (DataSource != null)
+        {
+        <DataChart Height="100%" Width="100%"
+                   GridMode="GridMode.BeforeSeries"
+                   ChartTitle="Company Finances by Department"
+                   Subtitle="Wind Speed vs. Boat Speed"
+                   TitleTopMargin="10"
+                   IsHorizontalZoomEnabled="true"
+                   IsVerticalZoomEnabled="true"
+                   Brushes="#9FB328, #7446B9"
+                   Outlines="#9FB328, #7446B9"
+                   Legend="@LegendRef"
+                   @ref="DataChartRef">
+
+            <CategoryAngleAxis 
+                   Name="angleAxis"
+                   Label="Department"
+                   DataSource="@DataSource" />
+            <NumericRadiusAxis 
+                   Name="radiusAxis"
+                   InnerRadiusExtentScale=0.1
+                   MinimumValue=0 />
+        </DataChart>
+        }
+    </div>
+</div>
+
+@code {
+
+    private List<SampleRadialItem> DataSource;
+    private DataChart dataChart;
+    private DataChart DataChartRef
+    {
+        get { return dataChart; }
+        set
+        {
+            dataChart = value;
+            this.setSeries("Pie");
+            SeriesType = "Pie";
+            StateHasChanged();
+        }
+    }
+
+    private Legend legend;
+    private Legend LegendRef
+    {
+        get { return legend; }
+        set
+        {
+            legend = value;
+            StateHasChanged();
+        }
+    }
+    private string seriesType;
+    private string SeriesType
+    {
+        get { return seriesType; }
+        set
+        {
+            seriesType = value;
+            StateHasChanged();
+        }
+    }
+
+    protected override void OnInitialized()
+    {
+        DataChartCoreModule.Register(IgniteUIBlazor);
+        DataChartRadialCoreModule.Register(IgniteUIBlazor);
+        DataChartRadialModule.Register(IgniteUIBlazor);
+        DataChartExtendedAxesModule.Register(IgniteUIBlazor);
+        DataChartCategoryModule.Register(IgniteUIBlazor);
+        DataChartInteractivityModule.Register(IgniteUIBlazor);
+
+        LegendModule.Register(IgniteUIBlazor);
+        DataSource = SampleRadialData.Create();
+    }
+
+    public void OnSeriesTypeChanged(ChangeEventArgs e)
+    {
+        var selectedSeries = e.Value.ToString();
+        SeriesType = selectedSeries;
+        this.setSeries(selectedSeries);
+    }
+
+    public void setSeries(string seriesType)
+    {
```

Comment:
there should be no code-behind to create series/axes and all these series/axes must be created in blazor code